e2e/tests/components/navigation/navigation.e2e-spec.ts

```diff
@@ -213,6 +227,48 @@ describe('Navigation Tests', () => {
         expect(await page.getPageContent()).toBe('Accounts Add');
     });
 
+    it('should disable node when disabled is to true', async () => {
+        const items = await page.getTopLevelItems();
+        await page.selectItem(items[3]);
+        const children = await page.getItemChildren(items[3]);
+
+        expect(await page.isItemDisabled(children[0])).toBeTruthy();
```

Comment:
```suggestion
        expect(await page.isItemDisabled(children[0])).toBe(true);
```
It doesn't matter in this context, but as a rule I recommend that you never use "fuzzy logic" in automated tests. Assert the exact value that you expect.